crds/tests/test_certify.py

```diff
@@ -800,6 +830,15 @@ def certify_rmap_compare():
     >>> test_config.cleanup(old_state)
     """
 
+def certify_roman_rmap_compare():
```

Comment:
Should there be a test where certifying against a context fails?

Reply:
@eslavich how do you feel about this? I don't have the best feel for regenerating "general case" type tests in CRDS.

Reply:
If we don't need it for the requirements, and it doesn't test anything that is specific to the `crds.roman` module, then I'm inclined not to duplicate the general tests.

Reply:
Skipped as duplicate.